docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -109,7 +160,38 @@ For the `DIS5K <https://xuebinqin.github.io/dis/index.html>`_ we prepared random
 Self-supervised Learning
 ************************
 
-To be added soon
+Self-supervised learning can be one of the solutions if the user has a small data set, but label information is not yet available.
+General self-supervised Learning in academia is commonly used to obtain well-pretrained weights from a source dataset without label information.
+However, in real-world industries, it is difficult to apply because of small datasets, limited resources, or training in minutes.
+
+For these cases, OTX provides improved self-supervised learning recipes that can be applied to the above harsh environments.
```

Comment:
Please, describe more here about algo components. Are they the same as for supervised incremental learning or not? ( I mean: optimizer, augmentations, learning rate scheduler, etc. )

Reply:
Self-SL has different components, but they are just for self-sl.
Self-sl training has two training stage (pretraining - fine tuning) and I added it.